src/Components/ProdDetail/OrderInformation/OrderInformation.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import './headOrderInformation.scss';
+import './bodyOrderInformation.scss';
+
+class OrderInformation extends Component {
+  render() {
+    const {
+      mainTitle,
+      price,
+      deliveryDateArr,
+      selectedDate,
+      quantity,
+      resultPrice,
+      className,
+      isSelectBoxOn,
+      isLike,
+      setDeliveryDate,
+      selectDeliveryDate,
+      subtractQuantity,
+      addQuantity,
+      closeSelectBox,
+      handleLike,
+      putCart,
+    } = this.props;
+
+    return (
+      <div className={className}>
+        <select
+          name="date"
+          onFocus={setDeliveryDate}
+          onChange={selectDeliveryDate}
+        >
+          <option value="">배송받을 날짜를 선택하세요</option>
+          {deliveryDateArr.map(date => {
+            return (
+              <option key={date.toString()} value={date}>
+                {date}
+              </option>
+            );
+          })}
+        </select>
+        <div
+          className="CartInformation"
+          style={!isSelectBoxOn ? { display: 'none' } : {}}
```

Comment:
이렇게도 가능하지만 가독성을 위해서 다음과 같이 하는 것이 더 좋을 것 같습니다.
```suggestion
          style={{ display: !isSelectBoxOn ? "none" : "block" }}
```

Reply:
넵, 감사합니다!